File:  problem_2.py
def decimal_to_binary(dec_num):
    if dec_num == 0:
        return "0"

    binary_num = ""

    while dec_num > 0:
        remainder = dec_num % 2
        binary_num = str(remainder) + binary_num
        dec_num = dec_num // 2

    return binary_num

user_Input = int(input("Input: "))
binary_Output = decimal_to_binary(user_Input)
print(f"Output: {binary_Output}")

File: problem_2.java
import java.util.Scanner;

public class CnvrtDToB {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Input: ");
        int decNum = s.nextInt();
        s.close();

        String binNum = CnvrtDToB(decNum);
        System.out.println("Output: " + binNum);
    }

    public static String CnvrtDToB(int decNum) {
        StringBuilder binNum = new StringBuilder();

        while (decNum > 0) {
            int rmnd = decNum % 2;
            binNum.insert(0, rmnd);
            decNum = decNum / 2;
        }

        return binNum.toString();
    }
}

